buildsystem/python.cmake

```diff
@@ -431,32 +457,36 @@ function(python_finalize)
 		COMMENT "creating in-place modules"
 	)
 	add_custom_target(inplacemodules ALL DEPENDS "${INPLACEMODULES_TIMEFILE}")
-
+#]==]
 
 	# cleaning of all in-sourcedir stuff
```

Comment:
we should adapt all those below so we no longer support any in-sourcedir stuff

Reply:
I changed everything except `check_py_file_list` (which I believe works as-is);
The `clean*` targets are a best-effort mechanism to selectively remove files generated by various custom targets. We can drop them, if they are too high-maintenance.
There might be other files that need to be changed. (Pointers welcome.)